rp2040-hal/src/rtc.rs

```diff
@@ -1,3 +1,620 @@
 //! Real Time Clock (RTC)
 // See [Chapter 4 Section 8](https://datasheets.raspberrypi.org/rp2040/rp2040_datasheet.pdf) for more details
-// TODO
+
+pub use crate::clocks::RtcClock;
```

Comment:
`RtcClock` in this context means the oscillator which drives the RTC IP block on the chip. This module should define its own `Rtc` object, which owns a `pac::RTC` structure - see `Spi::new()` for an example.

```
let r = hal::rtc::Rtc::new(p.RTC);
// or
let r = p.RTC.split();
```

I prefer the former.